e2e-test/cbas_azure_e2e_test.py

```diff
@@ -170,92 +169,108 @@ def check_submission_status(cbas_url, method_id, run_set_id):
-# Check that CBAS is ready; if not exit the test after 10 minutes of polling
-logging.info(f"Polling to check if WORKFLOWS app is ready in workspace {workspace_id}...")
-cbas_url = poll_for_app_url(workspace_id, 'WORKFLOWS_APP', 'cbas', azure_token, leo_url)
-if cbas_url == "":
-    logging.error(f"WORKFLOWS app not ready or errored out for workspace {workspace_id}")
-    exit(1)
-
-# Create CROMWELL_RUNNER app in workspace
-create_app(workspace_id, leo_url, 'CROMWELL_RUNNER_APP', 'USER_PRIVATE', azure_token)
-
-# check that Cromwell Runner is ready; if not exit the test after 10 minutes of polling
-logging.info(f"Polling to check if CROMWELL_RUNNER app is ready in workspace {workspace_id}...")
-cromwell_url = poll_for_app_url(workspace_id, 'CROMWELL_RUNNER_APP', 'cromwell-runner', azure_token, leo_url)
-if cromwell_url == "":
-    logging.error(f"CROMWELL_RUNNER app not ready or errored out for workspace {workspace_id}")
-    exit(1)
-
-# check that WDS is ready; if not exit the test after 10 minutes of polling
-logging.info(f"Polling to check if WDS app is ready to upload data for workspace {workspace_id}...")
-wds_url = poll_for_app_url(workspace_id, 'WDS', 'wds', azure_token, leo_url)
-if wds_url == "":
-    logging.error(f"WDS app not ready or errored out for workspace {workspace_id}")
-    exit(1)
-
-# upload data to workspace
-upload_wds_data_using_api(wds_url, workspace_id, "e2e-test/resources/cbas/cbas-e2e-test-data.tsv", "test-data")
-
-# create a new method
-method_id = create_cbas_method(cbas_url, workspace_id)
-method_version_id = get_method_version_id(cbas_url, method_id)
-
-# submit workflow to CBAS
-run_set_id = submit_no_tasks_workflow(cbas_url, method_version_id)
-
-# Without polling CBAS, check if outputs were written back to WDS. We don't poll CBAS first to verify
-# that the callback API is working
-# Note: When "Multi-user Workflows: Auto app start up" phase is completed, CROMWELL_RUNNER app will
-#       be deployed automatically by WORKFLOWS app. As a result, a submission would take time to reach a
-#       terminal state as CROMWELL_RUNNER app would take a while to provision and then run the workflow.
-#       To avoid test failures when this happens we poll for 10 minutes to check if outputs are written back to WDS.
-logging.info("Polling to check if outputs were successfully written back to WDS...")
-poll_for_outputs_data(wds_url, workspace_id, 'test-data', '89P13')
-
-# check submission status
-logging.info("Checking submission status...")
-check_submission_status(cbas_url, method_id, run_set_id)
-
-# delete workspace and apps
-# TODO: Enable workspace deletion check after bug https://broadworkbench.atlassian.net/browse/WOR-1309 is fixed
-# logging.info("Starting workspace deletion...")
-# delete_workspace(billing_project_name, workspace_name, rawls_url, azure_token)
-
-logging.info("Test completed successfully.")
+found_exception = False
```

Comment:
Note to reviewers: There are lot of whitespaces changes in L194-L262. One can turn on `Hide whitespaces` to make it easier to see what are the actual changes in this block.